.github/workflows/build.yaml

```diff
@@ -0,0 +1,22 @@
+name: "Build"
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - run: npm ci
+      - run: npm run test:unit # Can't run the UI tests without a UI
+      - run: npm install -g @vscode/vsce
+      - run: vsce package
+      - name: Publish VS Code extension artifact
```

Comment:
I don't think there is a need to package and publish in this action. Running the tests are fine.

I am thinking around the lines of a separate action that package and publish the extension artifact to vs code market place and github release.

Reply:
It's not a publish so much as it is 'attach this binary to the action so that you could download and run it if you fancy'.

Whilst we're making schema changes, that's less relevant. If there were a PR making functional changes, easier to grab the blob and run it than to check out and build and package and run?

There's definitely an argument to be made for lots of energy being burned for a once-in-a-while-maybe-never event, mind...